h/accounts/test/models_test.py

```diff
@@ -6,20 +6,21 @@
 import pytest
 from sqlalchemy import exc
 
+from h import db
 from h.accounts import models
 from h.test import factories
 
 
-def test_activation_has_asciinumeric_code(db_session):
```

Comment:
I'm not sure about the change to autouse the `database_session` fixture. Passing it in as an argument made it easier IMO to trace the setup/teardown code that happens around each test.